common/main/gl.xml

```diff
@@ -9797,27 +9797,26 @@ Warnings: All cp values have U+FE0F characters removed. See /annotationsDerived/
 		<personName order="sorting" length="long" usage="referring" formality="formal">
 			<namePattern>{surname-core} {surname2}, {given} {given2} {surname-prefix}</namePattern>
 			<namePattern alt="1">{surname-core}, {given} {given2} {surname-prefix}</namePattern>
-			<namePattern alt="2">↑↑↑</namePattern>
 		</personName>
 		<personName order="sorting" length="long" usage="referring" formality="informal">
 			<namePattern>{surname-core} {surname2}, {given-informal} {surname-prefix}</namePattern>
-			<namePattern alt="1">↑↑↑</namePattern>
+			<namePattern alt="1">{surname-core}, {given-informal} {surname-prefix}</namePattern>
 		</personName>
 		<personName order="sorting" length="medium" usage="referring" formality="formal">
 			<namePattern>{surname-core} {surname2}, {given} {given2-initial} {surname-prefix}</namePattern>
-			<namePattern alt="1">{surname-core} {surname2}, {given} {given2} {surname-prefix}</namePattern>
+			<namePattern alt="1">{surname-core}, {given} {given2} {surname-prefix}</namePattern>
```

Comment:
Did you want this to be `{given2}` when the first entry has `{given2-initial}`?

Reply:
Missed this, will examine in 1/2 hour

Reply:
Got it (lined up all the others for comparison, too, so I wouldn't miss any others). Good catch.